examples/opentelemetry/otel-collector-config.yaml

```diff
@@ -21,18 +22,17 @@ processors:
     check_interval: 5s
 
 exporters:
-  logging:
-    loglevel: debug
+  debug:
 
 service:
   pipelines:
     traces:
       receivers: [otlp]
       processors: [memory_limiter, batch]
-      exporters: [logging]
+      exporters: [debug]
```

Comment:
im not sure we want this - but also not sure iiuc the implications

Reply:
A renaming in the Otel collector that happens a few versions ago.

Reply:
<img width="806" alt="Screenshot 2024-05-02 at 14 31 07" src="https://github.com/envoyproxy/envoy/assets/14020598/e74b546e-987d-466c-b728-e712a9a33092">

https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter

Reply:
ok, thanks for explanation